integration/setup/config.go

```diff
@@ -1,16 +1,174 @@
 package setup
 
 import (
+	"github.com/giantswarm/apprclient"
+	"github.com/giantswarm/aws-operator/integration/env"
+	"github.com/giantswarm/backoff"
 	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/e2e-harness/pkg/framework/filelogger"
 	"github.com/giantswarm/e2e-harness/pkg/framework/resource"
-	awsclient "github.com/giantswarm/e2eclients/aws"
+	"github.com/giantswarm/e2e-harness/pkg/release"
+	e2eclientsaws "github.com/giantswarm/e2eclients/aws"
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
 )
 
+const (
+	namespace    = "giantswarm"
+	organization = "giantswarm"
+	quayAddress  = "https://quay.io"
+)
+
 type Config struct {
-	AWSClient *awsclient.Client
+	AWSClient *e2eclientsaws.Client
 	Guest     *framework.Guest
 	Host      *framework.Host
+	Release   *release.Release
 	Resource  *resource.Resource
 	Logger    micrologger.Logger
 }
+
+func NewConfig() (Config, error) {
+	var err error
+
+	var awsClient *e2eclientsaws.Client
+	{
+		awsClient, err = e2eclientsaws.NewClient()
+
+		if err != nil {
+			return Config{}, microerror.Mask(err)
+		}
+	}
+
+	var logger micrologger.Logger
+	{
+		c := micrologger.Config{}
+
+		logger, err = micrologger.New(c)
+		if err != nil {
+			return Config{}, microerror.Mask(err)
+		}
+	}
+
+	var apprClient *apprclient.Client
```

Comment:
This is actually defaulted in `Release`, but still can't hurt to do it explicitly here.

Reply:
It hurts me :D I will remove.